apps/src/code-studio/levels/multi.js

```diff
@@ -203,6 +205,13 @@ Multi.prototype.ready = function () {
       resetButton.click(() => resetContainedLevel());
     }
   }
+
+  if (
+    this.selectedAnswers.length === this.numAnswers &&
```

Comment:
is it possible to use your `correctNumberAnswersSelected` from below here, or is it out of scope?

Reply:
I think it's possible and I just forgot to replace this instance!